content/launch-keyboard.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuring your new Launch keyboard
+description:
+  View, add, or modify keys in your OS.
+keywords:
+  - keyboard shortcuts
+  - launch
+  - pop
+  - ubuntu
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+---
+
+## The Launch and Accessories
+
+The following items are included with your Launch:
+
+![Accessories](/files/launch-keyboard/launch-accessories.webp)
+
+## QMK firmware
+
+The Launch keyboard uses our version [QMK firmware](https://github.com/system76/qmk_firmware) and it is built using our changes that are not in upstream [QMK firmware](https://github.com/qmk/qmk_firmware). You can read more about QMK [here](https://github.com/system76/qmk_firmware).
+
+## Layers
+
+The core idea of "Layers" is that one key can be used in different setups for example on our laptops holding the <kbd>Fn</kbd> then pressing the <kbd>F5</kbd> will lower the volume while pressing <kbd>F5</kbd> on it's own will perform a different action like refreshing a web page if you are in a web browser like <u>Firefox</u>. You can read more about layers [here](https://beta.docs.qmk.fm/using-qmk/software-features/feature_layers
+).
+
+## Firmware Updates
+
+Firmware updates for the Launch are handled using [fwupd](https://fwupd.org/) and our <u>Firmware Manager</u> can be used to apply the update. When you get a notification about a firmware update you will need to unlock the device using Fn+Esc (note that this will be the default keys even if you remapped them in the Keyboard Configurator) before you can apply the update. When the keyboard is unlocked you will see the following keys light up:
+
+- U N L O C K E D
+
+Once the keyboard is in this mode you can apply the firmware update, when the update is complete you will need to unplug and re-plug it in to exit this mode.
+
+## Keyboard Configurator
+
+You can download the `AppImage` for Ubuntu and other OSes and use the <u>Pop!\_Shop</u> for Pop!\_OS. Use this [link](https://github.com/pop-os/keyboard-configurator/releases
+) to download the application.
+
+You can also download it from the Arch Linux AUR. The main version is built from latest releases which is recommended for most users which you can find [here](https://aur.archlinux.org/packages/keyboard-configurator/
```

Comment:
> You can also download it from the Arch Linux AUR. The main version is built from latest releases which is recommended for most users which you can find [here]

This is a run-on clause. Each "which" phrase should have a comma before it, and it would be better if there weren't two in a row.